stream-applications-release-train/stream-applications-docs/src/main/asciidoc/overview.adoc

```diff
@@ -61,90 +61,6 @@ docker pull springcloudstream/cassandra-sink-kafka
 
 The preceding command pulls the latest Docker image of the _Cassandra sink_ with the Kafka binder.
 
-=== Build
```

Comment:
This content is for the stream apps ref doc which does not need to show the steps to build the repo locally. That info is already in the repo README.adoc and this content was out of date (e.g. referencing jib to build the docker images).